modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/portlet/action/UpdateKBArticleMVCActionCommand.java

```diff
@@ -92,13 +94,15 @@ protected void doTransactionalCommand(
 		String content = ParamUtil.getString(actionRequest, "content");
 		String description = ParamUtil.getString(actionRequest, "description");
 		String sourceURL = ParamUtil.getString(actionRequest, "sourceURL");
+		Date displayDate = ParamUtil.getDate(
+			actionRequest, "displayDate",
+			new SimpleDateFormat("yyyy-MM-dd HH:mm"));
```

Comment:
Can you use `DateFormatFactoryUtil` to create the date format? Look for other usages of ParamUtil.getDate.

Reply:
Done! See https://github.com/liferay-lima/liferay-portal/pull/4677/commits/7220fc8c6e660a140dca8e9d5fb7d56a9b7a29b6